recipes/mlpack/all/conandata.yml

```diff
@@ -1,4 +1,13 @@
 sources:
+  "4.6.0":
```

Comment:
Please limit the PR to only adding the most recent version unless there's strong reasons why users may not be able to use the latest, thanks!

Reply:
I've added only the latest one instead